tests/src/test/scala/cats/tests/ApplicativeErrorSuite.scala

```diff
@@ -61,7 +61,7 @@ class ApplicativeErrorSuite extends CatsSuite {
     assert(compileErrors("e2.attemptNarrow[Num]").nonEmpty)
 
     val e3: Either[List[T[String]], Unit] = List(Str).asLeft[Unit]
-    assert(compileErrors("e3.attemptNarrow[List[Str.type]]").nonEmpty)
+    //assertEquals(compileErrors("e3.attemptNarrow[List[Str.type]]"), "")
```

Comment:
This is an odd one, because when I try it in the console it gives me an error but munit returns an empty string here